main.go

```diff
@@ -78,10 +80,22 @@ func main() {
 		os.Exit(1)
 	}
 
+	cfg, err := config.GetConfig()
+	if err != nil {
+		setupLog.Error(err, "")
+		os.Exit(1)
+	}
+	kclient, err := kubernetes.NewForConfig(cfg)
+	if err != nil {
+		setupLog.Error(err, "")
+		os.Exit(1)
+	}
+
 	if err = (&controllers.OpenStackAnsibleEEReconciler{
-		Client: mgr.GetClient(),
-		Scheme: mgr.GetScheme(),
-		Log:    ctrl.Log.WithName("controllers").WithName("OpenStackAnsibleEE"),
+		Client:  mgr.GetClient(),
+		Scheme:  mgr.GetScheme(),
+		Kclient: kclient,
```

Comment:
Where is this intended to be used in the controller? Is the manager cached client not enough?

Reply:
Looks like it's used in lib-common with helper at some places[1], but not sure if this operator would use those. But probably better to make the helper consistent across operators.

[1] https://github.com/openstack-k8s-operators/lib-common/blob/master/modules/common/pod/pod.go#L42-L46

Reply:
/lgtm